applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+To overcome this, we will use the _blake2_ hash function to encode the following properties: a binary prefix _"modlpy/nftproxy"_, followed by parachain, collection, and token ID. Both collection and token ID are u32 types, and
+padding them with zeros to length 32 prevents overlap.
+The _AccountId32_ type is then used to construct the address, ensuring a unique address for each NFTAA.
+
+Implementing the _nftaa_ act as a proxy function is the third critical step.
+This function requires verifying that the caller is the owner of the token, constructing a custom derivative function,
+setting the derived account as the caller, and executing the original call.
+This mechanism ensures secure and efficient operations, allowing NFTAAs to function effectively within the blockchain
+ecosystem.
+
+<img src="https://raw.githubusercontent.com/decenzio/NFTAA-Polkadot/main/proposal/nftaa_seq_dia_proxy.svg" alt="Proposal Diagram" width="70%" height="30%">
+
+Ultimately, the proposed pallet will enable the creation of NFTAAs, setting attributes, and staking on the relay chain via
+the NFTAA.
+This innovative approach not only enhances the functionality and security of NFTs within the blockchain ecosystem but
+also opens new possibilities for their application in various business and financial contexts.
+By combining the strengths of NFTs and blockchain technology, NFTAA represents a significant step forward in the
+evolution of digital assets and their management.
+
+### Ecosystem Fit
+
+The NFTAA project aims to modernize staking and account management within the Polkadot and Substrate ecosystems through
+the innovative use of NFTs by leveraging the flexibility of the Substrate framework and Polkadot's interoperability features.
+These NFT-based accounts can own assets, participate in staking, and serve as proxies for other accounts, significantly
+enhancing security, transparency, and liquidity within blockchain networks.
+
+NFTAA targets a diverse audience, including parachain developers, dApp developers, wallet developers, end users, and
+researchers.
+Parachain developers building custom blockchain networks on Polkadot can benefit from enhanced staking and account
+management features offered by NFTAA.
+dApp developers will find NFTAA particularly useful for secure and flexible account management and staking solutions.
+Developers of cryptocurrency wallets can integrate NFTAA to provide users with advanced functionalities, thereby
+improving the user experience.
+Both individuals and enterprises looking for improved liquidity in their blockchain assets can leverage
+NFTAA for better asset management.
+Additionally, researchers and academics interested in exploring new blockchain innovations can study and expand upon the
+unique concepts introduced by NFTAA.
+
+The NFTAA project addresses several critical needs within the blockchain ecosystem.
+Utilizing NFTAA provides more methods for managing and delegating staking rights without exposing private keys.
```

Comment:
If we have 1000 DOT locked in staking and suddenly we need to sell them for some reason we have to wait 28 days. If we want to sell the account the only option is to sell my PK (unsecure).

If we have 1000 DOTs staked through NFTAA we can simply sell the NFTAA (probably for any compensation) and it will be safe for both parties.

Staking is just an example, there are a number of different examples regarding locks and assets.